platform/vs/docker-sonic-vs/start.sh

```diff
@@ -37,7 +37,31 @@ mkdir -p /var/run/redis/sonic-db
 cp /etc/default/sonic-db/database_config.json /var/run/redis/sonic-db/
 
 SYSTEM_MAC_ADDRESS=$(ip link show eth0 | grep ether | awk '{print $2}')
-sonic-cfggen -a '{"DEVICE_METADATA":{"localhost": {"mac": "'$SYSTEM_MAC_ADDRESS'", "buffer_model": "traditional"}}}' $CHASS_CFG --print-data > /etc/sonic/init_cfg.json
+cat <<EOF > /tmp/init_cfg.json
+{
+    "DEVICE_METADATA": {
+        "localhost": {
+            "mac": "$SYSTEM_MAC_ADDRESS",
+            "buffer_model": "traditional"
+        }
+    },
+    "FEATURE": {
+        "swss": {
+            "state": "enabled"
+        },
+        "bgp": {
+            "state": "enabled"
+        },
+        "teamd": {
+            "state": "enabled"
+        },
+        "nat": {
```

Comment:
@stepanblyschak how do you know that we need exactly these features? IMHO, better to avoid something hardcoded.